### DISCUSSION TITLE
Defaultvalue on "Many to Many" fields.

### DISCUSSION THREAD

**Discussion by FreeSynergy** [directus/directus#23712] [Feature Requests]
### Summary

I would like to make a default value when creating a data record. It is important that the value of the input is part of the table in which it was entered. An addition would be if, for example, the value is taken from a variable (should be better documented somewhere how to use variables and fields within filters) - if this does not exist, NULL can be returned.

### Basic Example

Example:
I have a Group table and a Members field there. The Members field is linked to the Directus_Users table as M:M. When I create a new data record in Group, I want the user to be saved in the Members field at the same time. I would do this with the dollar sign: $CURRENT_USER

### Motivation

Simplify inputs and enter preconditions at the same time. This is and can be normal if you want to create data records. Not only in the case of User, but also in other cases.

### Detailed Design

Input of Default Value.

If Default Value does not exist in the target table, then enter a NULL value (or, if necessary, indicate another field that can be used for a value - first value, random, NULL or something else).

### Must Have:
- Default Value
- Variables/Fields

### Could Have:
- alternate value input field

### Drawbacks

I don't know - because it's too complicated to program???

### Alternatives

Otherwise you have to program something from the outside that is actually standard in the database design ... which would make no sense ...

### Adoption Strategy

I don't see any side effects with this feature. If nothing is entered, everything stays the same. That's great, isn't it?

**Comment by jeremyzilar** [comment]
This would be great! At the moment, our editors have created a draft "event" page and are duplicating that when creating a new "event".

**Reply by FreeSynergy** [reply]
Yes … there are many workarounds … but why?
